_data/sites.json

```diff
@@ -10927,6 +10927,23 @@
         ]
     },
 
+    {
+        "name": "Moovit",
+        "url": "https://support.moovitapp.com/hc/requests/new",
+        "difficulty": "hard",
+        "notes": "Contact the customer support and request account deletion. For \"Feedback Type\", choose \"Personal Data/GDPR\".",
```

Comment:
Do they check that you live in a GDPR-covered country? If they require proof, then we should move to `limited`, if not, we can keep it as hard.

Reply:
I am waiting for them to reply and delete my account. Then I can confirm it.

Reply:
Ok!